resources/yki/queries.sql

```diff
@@ -1492,12 +1492,15 @@ SELECT
   es.language_code,
   es.level_code,
   ed.exam_date,
-  o.oid
+  o.oid,
+  oed.exam_date AS original_exam_date
```

Comment:
SQL queryn muutokset näyttäis oikealta, mutta en oo tähän kummoisemmin perehtynyt